src/script/components/calling/DeviceToggleButton.tsx

```diff
@@ -42,60 +42,60 @@ const DeviceToggleButton: React.FC<DeviceToggleButtonProps> = ({currentDevice, d
       data-uie-name="device-toggle-button"
       css={css`
         align-items: center;
-        flex-direction: column;
         color: #fff;
         cursor: pointer;
         display: inline-flex;
         ${styles};
       `}
     >
-      <div
-        role="button"
-        tabIndex={0}
-        className="device-toggle-button-indicator"
-        data-uie-name="device-toggle-button-indicator"
-        onClick={selectNextDevice}
-        onKeyDown={e => handleKeyDown(e, selectNextDevice.bind(null, e))}
-        css={{
-          display: 'flex',
-          marginTop: 8,
-        }}
-      >
-        {devices.map(device => {
-          const isCurrentDevice = device === currentDevice;
+      {devices.map(device => {
+        const isCurrentDevice = device === currentDevice;
 
-          return (
-            <span
-              key={device}
-              className="device-toggle-button-indicator-dot"
-              data-uie-name="device-toggle-button-indicator-dot"
-              data-uie-value={isCurrentDevice ? 'active' : 'inactive'}
-              css={{
-                '&:hover': {
-                  backgroundColor: isCurrentDevice ? 'var(--toggle-button-hover-bg)' : 'var(--app-bg)',
-                  border: isCurrentDevice
-                    ? '1px solid var(--toggle-button-hover-bg)'
-                    : '1px solid var(--toggle-button-unselected-hover-border)',
-                },
-                /* eslint-disable sort-keys-fix/sort-keys-fix */
-                '&:active': {
-                  /* eslint-enable sort-keys-fix/sort-keys-fix */
-                  backgroundColor: isCurrentDevice ? 'var(--accent-color)' : 'var(--toggle-button-unselected-bg)',
-                  border: '1px solid var(--accent-color)',
-                },
-                '&:not(:last-child)': {marginRight: 5},
-                backgroundColor: isCurrentDevice ? 'var(--accent-color)' : 'var(--app-bg-secondary)',
-                border: isCurrentDevice ? '1px solid var(--accent-color)' : '1px solid var(--foreground)',
-                borderRadius: '50%',
-                color: '#fff',
-                display: 'inline-block',
-                height: 10,
-                width: 10,
-              }}
-            />
-          );
-        })}
-      </div>
+        return (
+          <div
+            role="button"
```

Comment:
probably not related to this PR, but why can't we use native `<button>` element here? :D

Reply:
I don't see why not, I just assumed it was done that way for a reason previously. I can try it out